lib/natalie/compiler/binding_gen.rb

```diff
@@ -468,9 +468,24 @@ def generate_name
 gen.binding('Enumerator::ArithmeticSequence', 'size', 'Enumerator::ArithmeticSequenceObject', 'size', argc: 0, pass_env: true, pass_block: false, return_type: :Object)
 gen.binding('Enumerator::ArithmeticSequence', 'step', 'Enumerator::ArithmeticSequenceObject', 'step', argc: 0, pass_env: false, pass_block: false, return_type: :Object)
 
+
 gen.singleton_binding('ENV', 'inspect', 'EnvObject', 'inspect', argc: 0, pass_env: true, pass_block: false, return_type: :Object)
 gen.singleton_binding('ENV', '[]', 'EnvObject', 'ref', argc: 1, pass_env: true, pass_block: false, return_type: :Object)
 gen.singleton_binding('ENV', '[]=', 'EnvObject', 'refeq', argc: 2, pass_env: true, pass_block: false, return_type: :Object)
+gen.singleton_binding('ENV', 'store', 'EnvObject', 'refeq', argc: 2, pass_env: true, pass_block: false, return_type: :Object)
+gen.singleton_binding('ENV', 'assoc', 'EnvObject', 'assoc', argc: 1, pass_env: true, pass_block: false, return_type: :Object)
+gen.singleton_binding('ENV', 'fetch', 'EnvObject', 'fetch', argc: 1, pass_env: true, pass_block: false, return_type: :Object)
+gen.singleton_binding('ENV', 'delete', 'EnvObject', 'delete_key', argc: 1, pass_env: true, pass_block: false, return_type: :Object)
+gen.singleton_binding('ENV', 'key?', 'EnvObject', 'has_key', argc: 1, pass_env: true, pass_block: false, return_type: :bool)
+gen.singleton_binding('ENV', 'has_key?', 'EnvObject', 'has_key', argc: 1, pass_env: true, pass_block: false, return_type: :bool)
+gen.singleton_binding('ENV', 'include?', 'EnvObject', 'has_key', argc: 1, pass_env: true, pass_block: false, return_type: :bool)
+gen.singleton_binding('ENV', 'member?', 'EnvObject', 'has_key', argc: 1, pass_env: true, pass_block: false, return_type: :bool)
+gen.singleton_binding('ENV', 'to_h', 'EnvObject', 'to_hash', argc: 0, pass_env: true, pass_block: false, return_type: :Object)
+gen.singleton_binding('ENV', 'to_hash', 'EnvObject', 'to_hash', argc: 0, pass_env: true, pass_block: false, return_type: :Object)
+gen.singleton_binding('ENV', 'size', 'EnvObject', 'size', argc: 0, pass_env: true, pass_block: false, return_type: :Object)
+gen.singleton_binding('ENV', 'length', 'EnvObject', 'size', argc: 0, pass_env: true, pass_block: false, return_type: :Object)
+gen.singleton_binding('ENV', 'to_s', 'EnvObject', 'to_s', argc: 0, pass_env: false, pass_block: false, return_type: :Object)
+gen.singleton_binding('ENV', 'rehash', 'EnvObject', 'rehash', argc: 0, pass_env: false, pass_block: false, return_type: :Object)
```

Comment:
Let’s put this list of methods in alphabetical order.

Reply:
Done with latest push.

Reply:
Thanks!